.travis.yml

```diff
@@ -0,0 +1,14 @@
+# Travis configuration file
+language: ruby
+before_script:
+  - psql -c 'create database db_test;' -U postgres
+rvm: 
+  - "2.6.0"
+script:
+  - bundle install
+  - bundle exec rails db:create
+  - bundle exec rails db:migrate
+  - bundle exec rake spec
+  - bundle exec rubocop -a
```

Comment:
There is no point of running rubocop with `-a` flag. The autocorrected changes won't be commited. The goal is to see if check is clear or not.